src/router.ts

```diff
@@ -10,28 +10,54 @@
 
 'use strict';
 
-const config = require('config');
-const compression = require('compression');
-const Constants = require('./constants');
-const express = require('express');
-const jwtMiddleware = require('./jwt-middleware');
+import config from 'config';
+import compression from 'compression';
+import * as Constants from './constants';
+import express from 'express';
+import {middleware} from './jwt-middleware';
+import ActionsController from './controllers/actions_controller';
+import AdaptersController from './controllers/adapters_controller';
+import AddonsController from './controllers/addons_controller';
+import EventsController from './controllers/events_controller';
+import InternalLogsController from './controllers/internal_logs_controller';
+import LoginController from './controllers/login_controller';
+import LogsController from './controllers/logs_controller';
+import LogOutController from './controllers/log_out_controller';
+import NewThingsController from './controllers/new_things_controller';
+import NotifiersController from './controllers/notifiers_controller';
+import OAuthClientsController from './controllers/oauthclients_controller';
+import OAuthController from './controllers/oauth_controller';
+import PingController from './controllers/ping_controller';
+import PushController from './controllers/push_controller';
+import RootController from './controllers/root_controller';
+import SettingsController from './controllers/settings_controller';
+import ThingsController from './controllers/things_controller';
+import UpdatesController from './controllers/updates_controller';
+import UploadsController from './controllers/uploads_controller';
+import UsersController from './controllers/users_controller';
+import RuleEngine from './rules-engine/index';
+import ProxyController, {ProxyStore} from './controllers/proxy_controller';
+import ExtensionsController from './controllers/extensions_controller';
 const nocache = require('nocache')();
```

Comment:
Missed a couple `require()` calls.

Reply:
I had trouble with this one before.
But it seems to work now.